better_budget.py

```diff
@@ -36,19 +39,27 @@ def compare_budget(dataframes, budget, total):
 			print('Oops! In your {} budget, you have spent ${:.2f}, which is ${:.2f} over budget.'.format(key, spent, abs(saved)))
 	print('there is {} unaccounted for.'.format(total-total_spent))
 
-def plot_budget(dataframes, budget_dict):
+def plot_budget(dataframes, budget_dict, keyword_dict):
```

Comment:
Keywords is global so this is unnecessary.
Global is usually a bad thing, but since its behavior is expected to be constant this is okay.

Ideally keywords should be in a separate file with a function call `getKeywords()`. Constants are a bugger in python since they aren't natively supported, so we have to get creative with this.

Reply:
Daniel also mentioned in a verbal conversation about keywords in a separate file, I agree this should be the case! Thank you both for the edits.